stand/man/loader.efi.8

```diff
@@ -394,7 +394,7 @@ BootCurrent: 0001
 Timeout    : 2 seconds
 BootOrder  : 0000, 0001, 0003, 0004, 0005, 0006, 0001, 0008, 000A, 000B, 000C, 000E, 0007
 \&...
-+Boot0001* FreeBSD ZPOOL HD(1,GPT,b5d0f86b-265d-1e1b-18aa-0ed55e1e73bd,0x28,0x96000)/File(\EFI\FREEBSD\LOADER.EFI)
++Boot0001* FreeBSD ZPOOL HD(1,GPT,b5d0f86b-265d-1e1b-18aa-0ed55e1e73bd,0x28,0x96000)/File(\\EFI\\FREEBSD\\LOADER.EFI)
```

Comment:
No?

Reply:
What do you mean?

The current man-page renders like this:
```
+Boot0001* FreeBSD ZPOOL HD(1,GPT,b5d0f86b-265d-1e1b-18aa-0ed55e1e73bd,0x28,0x96000)/File(EEBSD
                                       nda0p1:/EFI/FREEBSD/LOADER.EFI /boot/efi//EFI/FREEBSD/LOADER.EFI
```

Are you suggesting that it's supposed to say `File(EEBSD)`?

I think it's supposed to display `File(\EFI\FREEBSD\LOADER.EFI)`.  In order to achieve that, the `\` need to be escaped.

Reply:
Huh, excuse me (now we're even 🤣). I didn't see that it's not rendering correctly.

You want `\e`.

Addl. Context: mandoc_char(7): `note that doubling it ('\\') is not the right way to output a backslash.`

Reply:
Aw, crap.  In pretty much every language, doubling-up a backslash is the way to print it.  There's 528 other `\\` in the man pages.  Easy to fix, but a pain to review.  I'll add that to my list.

Thanks for the fix!

Reply:
No, thank you!

Reply:
It's now rendering correctly. Sure wish we could get some Pp's after these .Ed's in the examples section. I think that's a syntax fix, but I don't object if you don't want to do it.

Reply:
I'm a bit leery of making that change, because it's a bit subjective.  For example, in
```
Copy the loader:

    # cp /boot/loader.efi /boot/efi/EFI/FREEBSD/LOADER.EFI
replacing the all caps part of the example with the proper path.
```

one change could be to remove the space before the example -- I mean, the "replacing" is clearly in the same "sentence" as the example.  (That's a style of writing that's relatively common in math and science, where an equation is "part of the same sentence".)

IMO, a better fix would be to write:
```
Copy the loader, after replacing the all-caps part of the example with the proper path:

    # cp /boot/loader.efi /boot/efi/EFI/FREEBSD/LOADER.EFI
```

... but that's getting into even more subjective judgement, rather than fixing objectively incorrect syntax.

I'll feel much more comfortable staying focused on objectively bad stuff (such as `\\`), and only move on to subjective writing after that's done.

Reply:
Yes, I agree that the second example is a better fix.